packages/logs/lib/client.ts

```diff
@@ -0,0 +1,77 @@
+import type { MessageRow, MessageRowInsert, MessageMeta, OperationRowInsert } from './types/messages.js';
+import { setRunning, createMessage, setFailed, setCancelled, setTimeouted, setSuccess } from './models/messages.js';
+import { getFormattedMessage } from './models/helpers.js';
+
+export class LogContext {
+    id: string;
+
+    constructor(opts: { parentId: string }) {
+        this.id = opts.parentId;
+    }
+
+    /**
+     * ------ Logs
+     */
+    async log(data: MessageRowInsert): Promise<void> {
+        await createMessage(getFormattedMessage({ ...data, parentId: this.id }));
+    }
+
+    async debug(message: string, meta: MessageMeta = null): Promise<void> {
+        await this.log({ type: 'log', level: 'debug', message, meta, source: 'nango' });
```

Comment:
source should be a variable. Would think maybe for enterprise customers this value would be different.

Reply:
source `nango` means internal system, better name is welcome, maybe `internal`?
log generated from sync/actions will be inserted with `log()`

Reply:
I like `internal` better than `nango`

Reply:
okay perfect then